change/react-native-windows-a9e67774-3a7f-4de3-94c2-855154ee0ce1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
This needs to be set to "patch" instead of "none" to force a new build to be generated and show up in changelogs.

Reply:
updated